test/codeu/chat/util/CommandTokenizerTest.java

```diff
@@ -0,0 +1,24 @@
+package codeu.chat.util;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class CommandTokenizerTest {
+
+	@Test
+	public void testTokenizer() {
+		CommandTokenizer quotesTokenizer = new CommandTokenizer(
+				"Command \"arg0\" \"arg1\" \"arg2\" \"arg3\"");
```

Comment:
Similarly, you might want to add a test for when arguments inside quotation marks contain spaces.